fabs/README.md

```diff
@@ -0,0 +1,156 @@
+# FloatingActionButton component
+
+## FloatingActionButton design specs
+
+You can find the design specs on [www.decathlon.design](https://www.decathlon.design)
+
+## Usage with Vitamin Theme
+
+By inheriting the Vitamin theme in your app, you just have to use the default `FloatingActionButton` component or the `VitaminFloatingActionButton*` component.
```

Comment:
`VitaminFloatingActionButton*` The '*' is there on purpose ?

Reply:
Sniper ! Error from me, I will fix it

Reply:
Will be fixed in the next PR update